qp_qiime2/tests/test_qiime2.py

```diff
@@ -76,6 +79,114 @@ def test_not_analysis_artifact(self):
         self.assertFalse(success)
         self.assertEqual(msg, 'Artifact "5" is not an analysis artifact.')
 
+    def test_feature_classifier(self):
+        dbpath = join(self.basedir, '..', '..', 'databases',
+                      'gg-13-8-99-515-806-nb-classifier.qza')
+        original_params = {
+            'qp-hide-method': 'classify_sklearn',
+            'qp-hide-plugin': 'feature-classifier',
+            'The feature data to be classified.': '8',
+            'qp-hide-paramDirection of reads with respect to reference '
+            'sequences. same will cause reads to be classified unchanged; '
```

Comment:
same -> Same?

Reply:
This comes from Q2, if you feel strongly about it, please open an issue directly in the plugin feature-classifier.